tests/unit/test_args.py

```diff
@@ -63,9 +67,11 @@ def _get_service_model(self, service_name=None):
         service_model = mock.Mock(ServiceModel)
         service_model.service_name = service_name
         service_model.endpoint_prefix = service_name
+        service_model.protocol = 'query'
+        service_model.protocols = ['query']
         service_model.metadata = {
             'serviceFullName': 'MyService',
-            'protocol': 'query',
```

Comment:
Why did we remove this line?  They both exist, right?

Reply:
They do :) it's just no longer necessary for the test, so it's one less line we need in the codebase.  It doesn't make a real difference to me, so I'm happy to add it back

Reply:
The reason I'm calling it out is you added `service_model.protocol` on line 70. If that's required, we should be keeping the metadata in sync. Otherwise we're creating surface area for future bugs unnecessarily.